### DISCUSSION TITLE
Allowing main element as well as main role

### DISCUSSION THREAD

**Discussion by nicholasgriffintn** [bbc/bbc-a11y#372] [Feature Requests]
Allow `<main>` as well as `<element role="main'>` as this is even more valid.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role

> As discussed / raised in https://github.com/bbc/bbc-a11y/pull/336

**Comment by votemike** [comment]
It seems like that is because <main> is an HTML 5 element that IE11 doesn't understand (and therefore doesn't apply the role to).
IE11 is not unsupported and so I see no need to enforce this rule.
